news/4.37/pde.md

```diff
@@ -57,7 +57,22 @@ After that just run your tests as the usual plugin test, you can find an example
 ## API Tools
 -->
 
-<!--
----
-## PDE Compiler 
--->
+
+## PDE Compiler
```

Comment:
Please add a ### subheading describing the new feature like for these other ones:

<img width="308" height="133" alt="image" src="https://github.com/user-attachments/assets/87ee56ac-5429-426f-8533-069c355f33ec" />